python/katana/dynamic_bitset.pyx

```diff
@@ -0,0 +1,56 @@
+from .cpp.libgalois.DynamicBitset cimport DynamicBitset as CDynamicBitset
+
+
+cdef class DynamicBitset:
+
+    cdef CDynamicBitset underlying
+
+    def __cinit__(self):
+        self.underlying = CDynamicBitset()
+
+    def resize(self, n):
+        """Resizes the bitset."""
+        self.underlying.resize(n)
+
+    def reserve(self, n):
+        """Reserves capacity for the bitset."""
+        self.underlying.reserve(n)
+
+    def clear(self):
+        """Clears the bitset."""
+        self.underlying.clear()
+
+    def shrink_to_fit(self):
+        """Shrinks the allocation for bitset to its current size."""
+        self.underlying.shrink_to_fit()
+
+    def size(self):
+        """Gets the number of bits held by the bitset."""
+        return self.underlying.size()
```

Comment:
should be `__len__` so it is accessed using the `len` function. This makes this act as a collection as people would expect. https://docs.python.org/3/reference/datamodel.html#object.__len__

Reply:
done